lib/PuppeteerSharp/Target.cs

```diff
@@ -92,8 +92,8 @@ public Task<Page> PageAsync()
 
         private async Task<Page> CreatePageAsync()
         {
-            var session = await _sessionFactory().ConfigureAwait(false);
-            return await Page.CreateAsync(session, this, Browser.IgnoreHTTPSErrors, !Browser.AppMode, Browser.ScreenshotTaskQueue).ConfigureAwait(false);
+            var session = await _sessionFactory().ConfigureAwait(false);
+            return await Page.CreateAsync(session, this, Browser.IgnoreHTTPSErrors, !Browser.SetDefaultViewport, Browser.ScreenshotTaskQueue).ConfigureAwait(false);
```

Comment:
It should be be `Browser.SetDefaultViewport` (remove the not) here 
https://github.com/kblok/puppeteer/blob/51645932b7dc3c3f3d958deb667c22b3519bcc91/lib/Target.js#L43